### PR TITLE
Added convenience methods to TableViewHeaderFooterView

### PR DESCRIPTION
**Pull request by sophialee0416**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Added obj-c convenience methods to TableViewHeaderFooterView setups.

### Verification

Manual testing through sample data.

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/301)

### CODE REVIEW

File: ios/FluentUI/Table View/TableViewHeaderFooterView.swift [hunk 1]
```diff
@@ -280,6 +280,10 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         NotificationCenter.default.addObserver(self, selector: #selector(handleContentSizeCategoryDidChange), name: UIContentSizeCategory.didChangeNotification, object: nil)
     }
 
+    @objc open func setup(style: Style, title: String, accessoryButtonTitle: String = "") {
```
Comment (rdeassis) [inline ios/FluentUI/Table View/TableViewHeaderFooterView.swift:283]: Could you add Xcode style documentations to these setup method overloads that are exposed to objc?

**Review by rdeassis — COMMENTED** [review]
The changes look good. Could you just add the Xcode documentation to those existing setup methods as well as the overloads you're adding?

File: ios/FluentUI/Table View/TableViewHeaderFooterView.swift (hunk 1 above)
Comment (harrieshin) [reply ios/FluentUI/Table View/TableViewHeaderFooterView.swift:283]: also the simpliest variation we care about might be just style and title ? what is most used variation in office?

Comment (rdeassis) [reply ios/FluentUI/Table View/TableViewHeaderFooterView.swift:291]: It seems like all the usage is indeed only style and title. I agree that providing another overload with these 2 params could be very useful.

**Review by rdeassis — APPROVED** [review]

**Pull request merged**